scouting.lic

```diff
@@ -49,10 +49,14 @@ class Scouting
 
   def take_trail(room_id)
     walk_to(room_id)
-    bput('scout trail', 'You notice a trailmarker')
-    return if 'I could not find what you were referring to' == bput('go trail', 'You set off into the wild', 'That would be difficult', 'I could not find what you were referring to')
-    waitfor('As your journey ends')
-    pause 0.5 until Room.current.id
+    case bput('scout trail', 'You notice a trailmarker', 'Because your account is free')
+    when 'You notice a trailmarker'
+      return if 'I could not find what you were referring to' == bput('go trail', 'You set off into the wild', 'That would be difficult', 'I could not find what you were referring to')
+      waitfor('As your journey ends')
+      pause 0.5 until Room.current.id
+    when 'Because your account is free'
```

Comment:
I don't think you need this `when` case since it's already the end of the method.

I suggest something like `return if 'Because your... == bput('scout...')'` and then something similar to the old code. What do you think?